src/controller.cpp

```diff
@@ -53,9 +55,15 @@ geometry_msgs::msg::TwistStamped Controller::computeVelocityCommands(
   const geometry_msgs::msg::Twist & robot_speed,
   nav2_core::GoalChecker * goal_checker)
 {
+
+  auto t_start = std::chrono::high_resolution_clock::now();
   nav_msgs::msg::Path transformed_plan = path_handler_.transformPath(robot_pose);
   geometry_msgs::msg::TwistStamped cmd =
     optimizer_.evalControl(robot_pose, robot_speed, transformed_plan, goal_checker);
+  auto t_end = std::chrono::high_resolution_clock::now();
+  double elapsed_time_ms = std::chrono::duration<double, std::milli>(t_end - t_start).count();
+
+  RCLCPP_INFO_THROTTLE(logger_, *clock_, 1000, "Control evaluation time %f", elapsed_time_ms);
```

Comment:
I don't think this is good to be spamming the terminal, maybe a DEBUG log or remove?